<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CledgerReportDlg</class>
 <widget class="QDialog" name="CledgerReportDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1457</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="m_csvBtn">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>580</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pdfBtn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>580</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>580</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="m_textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>174</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDateEdit" name="m_fromDate">
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="m_toDate">
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>411</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="m_DeptBox"/>
    </item>
    <item>
     <widget class="QComboBox" name="m_accountBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>461</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="m_nameFilter">
      <property name="placeholderText">
       <string>Filter name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_deptInCSV">
      <property name="text">
       <string>CSV with Department</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_individualLedger">
      <property name="text">
       <string>Individual Ledger</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
